src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,80 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.OptionalInt;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
+    private static final String COMA_REGAX = ",";
+    private static final int SUBTRACT = 1;
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        OptionalInt integer = numbers.stream()
+                .flatMap(number -> Arrays.stream(number.split(COMA_REGAX)))
+                .map(Integer::parseInt)
+                .filter(number -> number % 2 == 0)
+                .mapToInt(Integer::intValue)
```

Comment:
Use 1 map instead of 2 
```suggestion
                .mapToInt(Integer::parseInt)
                .filter(number -> number % 2 == 0)
```